Resources/Prototypes/Polymorphs/polymorph.yml

```diff
@@ -94,3 +94,12 @@
   revertOnCrit: false
   revertOnDeath: true
   revertOnEat: true
+
+- type: polymorph
+  id: TreeMorph
+  entity: FloraTree01
+  name: tree
```

Comment:
nothing even uses the "name" field for polymorph prototypes so it should just be removed.

Reply:
Removed